src/components/DeviceDetails/index.tsx

```diff
@@ -53,44 +54,53 @@ const DeviceDetails: React.FC<Props> = ({
       <div className="device-details__body">
         <div className="device-details__table">
           <div className="device-details__header-row device-details__row">
-            <div className="device-details__row-label">Apparaat</div>
+            <div className="device-details__row-label">{isAreaCamera ? 'Gebied' : 'Apparaat'}</div>
           </div>
           <div className="device-details__row">
             <div className="device-details__row-label">Categorie</div>
-            <div className="device-details__row-element">{category}</div>
-            <button
-              type="button"
-              className="device-details__question-mark-button"
-              onClick={() => {
-                history.push('/categories');
-              }}
-            >
-              <QuestionMarkIcon />
-            </button>
+            <div className="device-details__row-element">{isAreaCamera ? 'Camera toezichtsgebied' : device?.category}</div>
+
+            {!isAreaCamera &&
+              <button
+                type="button"
+                className="device-details__question-mark-button"
+                onClick={() => {
+                  history.push('/categories');
+                }}
+              >
+                <QuestionMarkIcon />
+              </button>
+            }
 
           </div>
-          {soort && (
+          {device?.soort && (
```

Comment:
why is this construction chosen? is dificult to read and hs no added value. can we restore the original construction.
moving the desctucturing in the begining of the function instead of the parameters would be easyer to read then in the parameters.

Reply:
if device.soort is undefined it will throw an error

Reply:
the device object may not be null, the component displays een **existing** object.

how about something like this in the beginning of the component?
```code
const { id, name, soort, category, privacy, contact, organisation, properties } = device || {};
```

Reply:
ja dat kan wel

Reply:
stom dat destructuring  niet werkt als er waardes undefined zijn